framework/src/st_things/things_stack/things_api.h

```diff
@@ -27,38 +27,257 @@
+ * @param result : 1 = published resource to cloud , 0 =failed to publish resource
+ *			to cloud.
+ * @param domain : cloud domain name.
+ * @param access_token : Contains information of cloud.
+ */
 typedef void (*things_cloud_con_result_func_type)(int result, char *domain, char *access_token);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback function for providing ownership transfer method change state.
+ * @param addr : address
+ * @param port : port number
+ * @param uuid : uuid
+ * @param event : Holds value of oic sec otm event.
+ */
 typedef void (*things_sec_otm_state_func_type)(const char *addr, uint16_t port, const char *uuid, int event);
+
+//--------------------------------------------------------------------------------
 typedef int (*things_stop_soft_ap_func_type)(int value);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback function for getting user's input confirmation for ownership transfer.
+ * @return : 1 in case of confirmed, otherwise return 0.
+ */
 typedef int (*things_get_user_confirm_func_type)(void);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback function for handling request message.
+ * @param p_resource : Instance of target resource.
+ * @return : success = 1, failure = 0.
+ */
 typedef int (*things_handle_request_func_type)(struct things_resource_s *p_resource);
+
+//--------------------------------------------------------------------------------
 typedef int (*things_update_dev_prov_data_func_type)(es_dev_conf_prov_data_s *p_dev_prov_data);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief callback function to get the request for user's confirmation for reset.
+ * @param func_carrier : callback for handling result of reset.
+ * @param reset_type   : Type of reset.It will be NEED confirmation for reset or auto reset.
+ * @return : success = 1, failure = 0.
+ */
 typedef int (*things_reset_confirm_func_type)(things_reset_result_func_type *func_carrier, things_es_enrollee_reset_e reset_type);
 
+//--------------------------------------------------------------------------------
+/**
+ * @brief Initialize the things stack.
+ * @param json_path	: Path to Json file which defines a thing. Definition includes
+ *				the device information,resources and their properties,
+ *				 configuration info for connectivity and easy-setup, etc.
+ * @easysetup_completed : Indicates whether easysetup is completed already or not.
+ * @return		: 1 on success, otherwise 0 value.
+ */
 int things_initialize_stack(const char *json_path, bool *easysetup_completed);
+
+//--------------------------------------------------------------------------------
+/**
+ * @brief Deinitialize the things stack.It delete all the device information.
```

Comment:
please white space between sentence and sentence.

Reply:
done.